system-contracts/scripts/utils.ts

```diff
@@ -253,3 +254,79 @@ export function prepareCompilerPaths(path: string): CompilerPaths {
 
   return new CompilerPaths(absolutePathSources, absolutePathArtifacts);
 }
+
+// Get the latest file modification time in the watched folder
+function getLatestModificationTime(folder: string): Date | null {
```

Comment:
add a comment on when it can return `null`

Reply:
comment added below